tvb_storage/tvb/storage/storage_interface.py

```diff
@@ -432,6 +426,17 @@ def export_simulator_configuration(self, burst, all_view_model_paths, all_dataty
 
         return result_path
 
+    def copy_datatypes(self, dt_path_list, data):
+        export_folder = self.__build_data_export_folder(data, self.EXPORT_FOLDER)
```

Comment:
This method holds similar code to lines 448-455 in `export_datatypes`. Could `export_datatypes` call this instead of keeping the same code in 2 places?

Reply:
Yes, it can, it is exactly the same code, nice catch.

Reply:
If you do the change related to `export_datatypes_structure`, than building the `export_folder` could move out of here and inside the `export_datatypes` method.